src/tests/ffi-key-sig.cpp

```diff
@@ -469,17 +472,47 @@ TEST_F(rnp_tests, test_ffi_export_revocation)
       rnp_ffi_set_pass_provider(ffi, ffi_string_password_provider, (void *) "wrong"));
     assert_rnp_failure(rnp_key_export_revocation(
       key_handle, output, 0, "SHA256", "superseded", "test key revocation"));
+    assert_rnp_failure(rnp_key_export_revocation(key_handle,
+                                                 output,
+                                                 RNP_KEY_EXPORT_ARMORED,
+                                                 "SHA256",
+                                                 "superseded",
+                                                 "test key revocation"));
+
+    rnp_output_destroy(output);
+    assert_rnp_success(rnp_key_handle_destroy(key_handle));
+    assert_rnp_success(rnp_ffi_destroy(ffi));
```

Comment:
With these changes we completely missing the case(s) when user attempts to successfully export revocation after few initial failures. So, again, please do not overwrite test cases but just extend them.